src/entities/calendaritem.cpp

```diff
@@ -486,12 +496,12 @@ bool CalendarItem::store() {
         query.prepare(
             "INSERT INTO calendarItem "
             "(summary, url, description, calendar, uid, related_uid, ics_data, "
-            "etag, last_modified_string, has_dirty_data, completed, "
+            "etag, last_modified_string, has_dirty_data, completed, progress, "
             "alarm_date, priority, created, modified, "
             "completed_date, sort_priority) "
             "VALUES (:summary, :url, :description, :calendar, :uid,"
             " :related_uid, :ics_data, :etag, :last_modified_string, "
-            ":has_dirty_data, :completed, :alarm_date, :priority, "
+            ":has_dirty_data, :completed, progress, :alarm_date, :priority, "
```

Comment:
This looks like a `:` is missing. Are you sure this query ever worked? 😉

Reply:
Oops.. I did not get any warnings though :joy: Added it there, will test again now.

Reply:
I don't see any noticeable difference but at least it's there